cdap-ui/app/features/hydratorplusplus/services/non-store-pipeline-error-factory.js

```diff
@@ -123,30 +123,6 @@ let hasValidName = (name, cb) => {
   cb(false);
 };
 
-let hasOnlyOneSource = (myHelpers, GLOBALS, nodes, cb) => {
-  let error;
-  let countSource = 0;
-  if (!myHelpers.objectQuery(nodes, 'length')) {
-    cb(false);
-  }
-  nodes.forEach( node => {
-    if (GLOBALS.pluginConvert[node.type] === 'source') {
-      countSource++;
-    }
-  });
-  if (countSource === 0) {
```

Comment:
Does this imply it won't throw an error if there is no source? Shouldn't check for >= 1 source?